src/app/index.js

```diff
@@ -85,13 +85,19 @@ function shapeDel(shape) {
 
 /** @param { CustomEvent<IMenuShapeDragOutEventDetail> } evt */
 function shapeAddingDragOut(evt) {
-	const point = svg.querySelector(`[data-templ='${evt.detail.shape}']`).getAttribute('data-center').split(',');
+	const selector = svg.querySelector(`[data-templ='${evt.detail.shape}']`);
+	const txtTags = [...selector.getElementsByTagName('text')];
```

Comment:
Thank you for your help. Not sure it is best idea to store deault text values in defs. II would prefer delte "Title" string from defs.